slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -736,6 +749,29 @@ class SlideToActView @JvmOverloads constructor (
         }
     }
 
+    /**
+     * Private method to handle vibration logic, called when the cursor it moved to the end of it's path
+     * @throws SecurityException if bumpVibration is greater than 0 but the permission
+     * */
+    @SuppressLint("MissingPermission")
+    private fun handleVibration() {
+        if (ContextCompat.checkSelfPermission(context, Manifest.permission.VIBRATE) !=
+                PackageManager.PERMISSION_GRANTED)
+
+            throw SecurityException(
+                    "bumpVibration is set but permissions are unavailable." +
+                    "You must have the permission android.permission.VIBRATE in " +
+                    "AndroidManifest.xml to use bumpVibration")
```

Comment:
* Missing curly braces

* Please change the logic here to:
  * If `bumpVibration > 0 && permission not granted` -> Print to stderr
  * If `bumpVibration > 0 && permission granted` -> Vibrate
  * else -> Do nothing